service/api.go

```diff
@@ -678,6 +758,8 @@ func (as *APIServer) GetServiceApp() *gimlet.APIApp {
 	app.Route().Version(2).Route("/task/{taskId}/expansions").Wrap(requireTask, requireHost).Handler(as.GetExpansions).Get()
 
 	// plugins
+	app.Route().Version(2).Prefix("/task/{taskId}").Route("/newPush").Wrap(requireTask).Handler(as.NewPush).Post()
+	app.Route().Version(2).Prefix("/task/{taskId}").Route("/updatePushStatus").Wrap(requireTask).Handler(as.UpdatePushStatus).Post()
```

Comment:
nit: can this route be update_push_status (and new_push)? There are a couple of outliers that snuck through but this is our general convention.